maltoolbox/__main__.py

```
@@ -51,21 +52,21 @@ def generate_attack_graph(
             log_configs['attackgraph_file']
         )
 
-    if send_to_neo4j:
```

Why uncomment this?

---

Because neo4j doesn't currently work anymore.

---

Got it. But I think you should instead raise an exception 'NEO4j does not work' so users don't go mental when their graphs don't show up.

---

I will one of us will fix it at some later point and I don't think enough people use Neo4j and keep their mal-toolbox up to date for it to matter.